src/components/calcite-accordion-item/calcite-accordion-item.scss

```diff
@@ -1,24 +1,19 @@
-//ie11 fixes
-:host-context([theme="dark"]) {
-  @include calcite-theme-dark();
-}
-
 // scale variants for child
-:host-context([scale="s"]) {
+:host([scale="s"]) {
```

Comment:
I would leave these as `:host-context` as accordions actually work pretty well in ie11 right now